client/app/components/conference_room/availability/RoomsAvailability.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ConferenceRoomSource from 'app/sources/ConferenceRoomSource';
+import AllDayAvailable from 'components/conference_room/availability/AllDayAvailable';
+import CurrentlyAvailable from 'components/conference_room/availability/CurrentlyAvailable';
+import CurrentlyBusy from 'components/conference_room/availability/CurrentlyBusy';
+import uuid from 'uuid';
+import { AVAILABILITY, sortByAvailability, buildAvailabilityProps } from 'helpers/AvailabilityHelper';
+
+const allDayAvailableComponent = (props) =>
+  <AllDayAvailable key={uuid()} conferenceRoomTitle={props.conferenceRoomTitle} />;
+
+const currentlyAvailableComponent = (props) =>
+  <CurrentlyAvailable key={uuid()} conferenceRoomTitle={props.conferenceRoomTitle} duration={props.duration} />;
+
+const currentlyBusyComponent = (props) =>
+  <CurrentlyBusy key={uuid()} conferenceRoomTitle={props.conferenceRoomTitle} duration={props.duration} />;
+
+export default class RoomsAvailability extends React.Component {
+  constructor(...args) {
+    super(...args);
+    this.state = { events: [], conferenceRooms: [] };
+    this._fetchConferenceRooms();
```

Comment:
I think it would be better if you just passed conferenceRooms as prop.
Especially that we already have those at the top level.